kotlinx-coroutines-debug/README.md

```diff
@@ -170,9 +170,6 @@ conflicts, but:
 * `kotlinx-coroutines-debug` transitively depends on JNA and JNA-platform, both of which include license files in their
   META-INF directories. Trying to merge these files leads to conflicts, which means that any Android project that
   depends on JNA and JNA-platform will experience build failures.
-* Additionally, `kotlinx-coroutines-debug` embeds `byte-buddy-agent` and `byte-buddy`, along with their resource files.
-  Then, if the project separately depends on `byte-buddy`, merging the resources of `kotlinx-coroutines-debug` with ones
-  from `byte-buddy` and `byte-buddy-agent` will lead to conflicts as the resource files are duplicated.
```

Comment:
At this point, I think one of two ways is more efficient than ping-ponging in this review.
Either we merge the contribution as is and you can quickly edit the readme to your liking in a new commit, or you might just want to edit the commit directly pre merge.

WDYT?

Reply:
There's also a third way, the one we usually take in this library: the contributor carefully looks at the places affected by the proposed change, considers the consequences, and then introduces sweeping, coordinated edits. In other words, instead of making a small isolated change, they consider the bigger picture. A reviewer's job then boils down to highlighting the affected areas the contributor hasn't thought of / didn't know about (not to enumerate every specific line of code to change), after which the contributor goes back to researching what the proposed change entails and introduces further sweeping edits. No ping-ponging results, as the list of such areas to consider is typically limited.

Although the third way is preferred, both of the options you've proposed are okay: we're also glad and thankful to accept incomplete contributions if the contributor says: "I'm no longer interested in working on this, you do it." In this scenario, the credit still goes to the contributor.

Reply:
I pushed an update, please provide feedback

Reply:
Nice, thanks! I've also left a comment with my review, it's possible that you've missed it:

> With this change, we are no longer shading anything, so could you please also remove all mentions of the shadow plugin from our library?

Reply:
Done ✅